apps/bluetooth/js/settings.js

```diff
@@ -27,7 +27,7 @@ navigator.mozL10n.once(function bluetoothSettings() {
     window.close();
   };
 
-  var gBluetoothCheckBox = document.querySelector('#bluetooth-status input');
+  var gBluetoothCheckBox = document.querySelector('.bluetooth-status gaia-switch');
```

Comment:
I'm just changing this due to adjusting the markup a bit to better-match the v2 markup - makes life easier when grepping/debugging.